app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -2,104 +2,102 @@
   background: govuk-colour("white");
   margin-top: govuk-spacing(6);
 
-  @include govuk-media-query($from: tablet) {
+  @include govuk-media-query($from: desktop) {
     margin-top: govuk-spacing(9);
-    border-bottom: 1px solid govuk-colour("white");
   }
 
+  // New design has the box flush with edges of smaller screens
+  // We need to compensate for `govuk-width-container` margins:
+  @include govuk-media-query($until: tablet) {
+    margin-right: -15px;
+    margin-left: -15px;
+  }
+  @include govuk-media-query($from: tablet, $until: desktop) {
+    margin-right: -30px;
+    margin-left: -30px;
+  }
   // Scoped to the feedback component temporarily
   [hidden] {
     // stylelint-disable-next-line declaration-no-important
     display: none !important;
   }
 }
 
-.gem-c-feedback__prompt-questions {
-  text-align: center;
-  border-bottom: 1px solid govuk-colour("white");
-  padding: govuk-spacing(5) govuk-spacing(5) govuk-spacing(5) govuk-spacing(5);
-  box-sizing: border-box;
-
+.gem-c-feedback__prompt {
+  background-color: govuk-colour("light-grey");
+  color: govuk-colour("black");
+  border-top: 1px solid govuk-colour("mid-grey");
+  outline: 0;
   @include govuk-media-query($from: tablet) {
-    width: 50%;
-    display: table-cell;
-    text-align: left;
-    border-bottom: 0;
+    @include govuk-font(16, $weight: bold);
```

Comment:
Just to check, is it definitely necessary to include this font definition? It seems that we're already setting various font sizes and weights in the rest of component styles.

Reply:
Good spot, this is unnecessary and now deleted.